src/App.js

```diff
@@ -31,6 +32,7 @@ function App() {
         <Route path="/wishlist" element={<WishListView />} />
         <Route path="/checkout/completed" element={<CheckoutComplete />} />
         <Route path="/profile" element={<UserProfile />} />
+        <Route path="/forgotPassword" element={<ForgotPassword />} />
```

Comment:
This path `forgotPassword` doesn't match the url that clicking the "Forgot Password" link in `signIn.js` points to (`forgot-password`).

So right now clicking the link sends me to a 404 page not found page:
![Uploading Screenshot 2023-11-08 at 9.40.09 AM.png…]()

Reply:
I have updated the url in App.js 👍 commit ed4f7706835248df881b08a14fdd86a1f419a598